portfolio/pom.xml

```diff
@@ -40,6 +40,14 @@
     </dependency>
   </dependencies>
 
+<!-- gson -->
+ <dependency>
```

Comment:
These should match the indentation of the `<dependency>` elements above. Once you make this file a valid XML file I think formatting the document will fix this for you.